PayPal/src/test/java/com/braintreepayments/api/paypal/PayPalVaultRequestUnitTest.java

```diff
@@ -301,57 +299,4 @@ public void createRequestBody_sets_userPhoneNumber_when_not_null() throws JSONEx
 
         assertTrue(requestBody.contains("\"phone_number\":{\"country_code\":\"1\",\"national_number\":\"1231231234\"}"));
     }
-
-    @Test
-    public void createRequestBody_sets_shippingCallbackUri_when_not_null() throws JSONException {
```

Comment:
Are we able to add these tests to `PayPalCheckoutRequestUnitTest`?

Reply:
Yes, there are already 3 unit tests that cover this in `PayPalCheckoutRequestUnitTest ` 
https://github.com/braintree/braintree_android/blob/388c2d64336e0a94ad83995f9ac0c7ffc4e49d32/PayPal/src/test/java/com/braintreepayments/api/paypal/PayPalCheckoutRequestUnitTest.java#L162